persistence-utils-demo-test/pom.xml

```diff
@@ -1,15 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.axonivy.ivy.api</groupId>
+    <artifactId>ivy-project-parent</artifactId>
+    <version>13.2.0-SNAPSHOT</version>
+    <relativePath></relativePath>
+  </parent>
   <groupId>com.axonivy.utils.persistence</groupId>
   <artifactId>persistence-utils-demo-test</artifactId>
-  <version>12.0.3-SNAPSHOT</version>
+  <version>13.2.0-SNAPSHOT</version>
   <packaging>iar</packaging>
   <description>Test project of the persistence utils demo project. Note, that this project is currently also used to test some aspects of the persistence library.</description>
   <properties>
-    <project.build.plugin.version>12.0.0</project.build.plugin.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <tester.version>12.0.1</tester.version>
+    <tester.version>13.2.0</tester.version>
+    <project.build.plugin.version>13.1.1</project.build.plugin.version>
```

Comment:
I'd use the 13.2.0-SNAPSHOT

Reply:
or maybe better to complete these properties altogether, the versions should now be defined by consuming the ivy-project-parent.

Reply:
Yes, I think that would be the better solution.